nimbus/Cargo.toml

```diff
@@ -20,7 +20,7 @@ serde_derive = "1"
 serde_json = "1"
 log = "0.4"
 # Viaduct locked to version 61.0.10 of Application Services
-viaduct = { git = "https://github.com/mozilla/application-services",  rev = "2478bcf2b48d1867b01e8b7df4f86a69d564d49a"}
+viaduct = { git = "https://github.com/mozilla/application-services",  rev = "5bbda246ec2ef14c3e4ee6dd37541b038d63d6f6"}
```

Comment:
This is the current head of appservices

Reply:
I thought we wanted to largely unpin app-services?  Eg put it at 61 or whatever the current head is?

Reply:
It's not released on crates.io, so we have to pin to *some* rev here, but we could use a tag rather than commit hash maybe?

Reply:
That at least communicates more clearly what the point of the pin is (i.e. does it really require that _exact_ rev, or...).

Reply:
yep, I'm going to try with a tag, thanks